File: picture-server/src/main/java/com/applory/pictureserver/domain/config/AppConfiguration.java
package com.applory.pictureserver.domain.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Configuration
@ConfigurationProperties(prefix = "picture")
@Data
public class AppConfiguration {
    private String pwSalt;

    private String jwtSignKey;

    private String clientId;

    private String clientSecret;
}

File: picture-server-kt/src/main/kotlin/com/applory/pictureserverkt/config/AppConfiguration.kt
package com.applory.pictureserverkt.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.ConstructorBinding

@ConfigurationProperties(prefix = "picture")
@ConstructorBinding
data class AppConfiguration(
    val pwSalt: String?,
    val jwtSignKey: String,
    val clientId: String,
    val clientSecret: String,
)
